include/openssl/ocsp.h

```diff
@@ -21,6 +21,33 @@ extern "C" {
 
 // Various OCSP flags and values
 
+// The following constants are OCSP reason codes identify the reason for the
+// certificate revocation.
+//
+//  CRLReason ::= ENUMERATED {
+//        unspecified             (0),
+//        keyCompromise           (1),
+//        cACompromise            (2),
+//        affiliationChanged      (3),
+//        superseded              (4),
+//        cessationOfOperation    (5),
+//        -- value 7 is not used
+//        certificateHold         (6),
+//        removeFromCRL           (8),
+//        privilegeWithdrawn      (9),
+//        aACompromise            (10) }
+//
+// Reason Code RFC: https://www.rfc-editor.org/rfc/rfc5280#section-5.3.1
+#define OCSP_REVOKED_STATUS_UNSPECIFIED 0
+#define OCSP_REVOKED_STATUS_KEYCOMPROMISE 1
+#define OCSP_REVOKED_STATUS_CACOMPROMISE 2
+#define OCSP_REVOKED_STATUS_AFFILIATIONCHANGED 3
+#define OCSP_REVOKED_STATUS_SUPERSEDED 4
+#define OCSP_REVOKED_STATUS_CESSATIONOFOPERATION 5
+#define OCSP_REVOKED_STATUS_CERTIFICATEHOLD 6
```

Comment:
Should we go ahead an define a constant for `7`? 
```c++
// The value 7 is not an assigned to a status
#define OCSP_UNASSIGNED_REVOKED_STATUS 7
```
Since this is the "internal" header, consumers won't have visibility to these constants.

Reply:
I had to move these to the external header, so they'll have visibility to the rest. But I'll add this to the internal header, definitely better than just using the magic number 7.